src/util.cpp

```diff
@@ -750,9 +750,7 @@ static boost::filesystem::path ZC_GetBaseParamsDir()
         pathRet = fs::path(pszHome);
 #ifdef MAC_OSX
     // Mac
-    pathRet /= "Library/Application Support";
-    TryCreateDirectory(pathRet);
-    return pathRet / "ZcashParams";
+    pathRet / "Library/Application Support/ZcashParams";
```

Comment:
@ch4ot1c for some reason I'm not seeing this change in master after your merge- was it removed intentionally?

Reply:
Yes; was fixed to Zcash's implementation:
```
    // Mac
    pathRet /= "Library/Application Support";
    TryCreateDirectory(pathRet);
    return pathRet / "ZcashParams";
```

Reply:
But certainly should instead be https://github.com/BTCPrivate/BTCP-Rebase/pull/36, good catch @donshin.